CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 --------------------------------------
 
     Version: 0.14.0
+    
+    - Breaking Changes:
+      * The order in which `FlaskView` decorators are applied has been reversed #49
```

Comment:
@hjc This is since v0.11.0, I don't think we should put it into v0.14.0, we should put into v0.11.0 section, is that right?

Reply:
Oh, didn't know that. Yea, lemme move it.